rskj-core/src/main/java/co/rsk/peg/whitelist/LockWhitelist.java

```diff
@@ -76,23 +82,51 @@ public Integer getSize() {
         return new ArrayList<>(whitelistedAddresses.keySet());
     }
 
-    public Coin getMaxTransferValue(Address address) {
-        return whitelistedAddresses.get(address);
+    public <T extends LockWhitelistEntry> List<T> getEntries(Class<T> type) {
+        return whitelistedAddresses.values().stream()
+                .filter(e -> e.getClass() == type)
+                .map(type::cast)
+                .collect(Collectors.toList());
+    }
+
+    public List<LockWhitelistEntry> getEntries() {
```

Comment:
Sorry, forgot this one :(
If you don't mind, maybe `getAll()`? Same for the overloaded one that filters?